README.md

```diff
@@ -59,12 +68,27 @@ export SAUCE_USERNAME=<username>
 export SAUCE_ACCESS_KEY=<access key>
 ```
 
-If `SAUCE_LABS` is not set, `karma` will only run the tests locally on PhantomJS.
+If `SAUCE_LABS` is not set, `karma` will only run the tests locally on Chrome.
 
 Browsers
 --------
 
-The tests run in the browsers specified in [launchers.json](https://github.com/resin-io-modules/resin-config-karma/blob/master/launchers.json).
+The local tests run in the headless Chrome.
+
+To run them locally you need Chrome >= 59 installed.
+
+To run them on Travis add the following lines to your `.travis.yml` file:
+
+```yaml
+dist: trusty
+sudo: false
```

Comment:
You're completely right, I've confused this with `sudo: required` (which [does](https://docs.travis-ci.com/user/reference/trusty/#Fully-virtualized-via-sudo%3A-required) slow down the build). This is totally fine and great :+1: